## This test is broken for several reasons:
## - It's linking a shared lib against a static lib.  This is not
##   portable, and we even give a warning about this.
## - The shared lib is not using any symbols from the static lib.
## - The test program is linked against the shared lib and using
##   symbols from the static lib.

Index: libtool-2.2.6a/tests/demo.at
===================================================================
--- libtool-2.2.6a.orig/tests/demo.at	2009-04-10 00:53:38.000000000 +0200
+++ libtool-2.2.6a/tests/demo.at	2009-04-10 00:53:48.000000000 +0200
@@ -436,6 +436,8 @@
 
 AT_SETUP([deplibs_check_method])
 
+AT_CHECK([(exit 77)])
+
 _LT_DEMO_SETUP
 
 AT_DATA([demo.mk],
